### PR TITLE
chore, refactor(egen): refactors the SDK_Custom_Training_Python_Package_Managed_Text_Dataset_Tensorflow_Serving_Container notebook

### PR DESCRIPTION
**Pull request by k-root**
- adds colab enterprise
- refactors code
- removes boilerplate
- formats according to template

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/sdk/SDK_Custom_Training_Python_Package_Managed_Text_Dataset_Tensorflow_Serving_Container.ipynb [hunk 1]
```diff
@@ -1335,20 +1116,20 @@
...
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "rxc_yVxkbbbd"
+        "id": "c6bb3da801b4"
       },
       "source": [
         "You need to specify the Python package that was built and uploaded to GCS, the module name of the Python package, the pre-built training container image uri for training, and in this example,  are using TensorFlow serving container for prediction."
```
Comment (kittyabs) [inline notebooks/official/sdk/SDK_Custom_Training_Python_Package_Managed_Text_Dataset_Tensorflow_Serving_Container.ipynb:1135]: - change: "pre-built" 
    - to: "prebuilt"

File: notebooks/official/sdk/SDK_Custom_Training_Python_Package_Managed_Text_Dataset_Tensorflow_Serving_Container.ipynb [hunk 2]
```diff
@@ -1422,7 +1203,7 @@
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "XO0-zdXxrKoo"
+        "id": "db696d27af44"
       },
       "source": [
         "# Deploy a Model and Create an Endpoint on Vertex AI\n",
```
Comment (kittyabs) [inline notebooks/official/sdk/SDK_Custom_Training_Python_Package_Managed_Text_Dataset_Tensorflow_Serving_Container.ipynb:1209]: change to:

- Deploy a model and create an endpoint on Vertex AI

File: notebooks/official/sdk/SDK_Custom_Training_Python_Package_Managed_Text_Dataset_Tensorflow_Serving_Container.ipynb [hunk 3]
```diff
@@ -1514,7 +1295,7 @@
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "gRkvzr2-LgA9"
+        "id": "60092cfdd571"
       },
       "source": [
         "# Batch Prediction Job on the Model"
```
Comment (kittyabs) [inline notebooks/official/sdk/SDK_Custom_Training_Python_Package_Managed_Text_Dataset_Tensorflow_Serving_Container.ipynb:1301]: - change: Batch Prediction Job on the Model
    - to: Batch prediction job on the model

**Review by kittyabs — COMMENTED** [review]
numerous edits

File: notebooks/official/sdk/SDK_Custom_Training_Python_Package_Managed_Text_Dataset_Tensorflow_Serving_Container.ipynb [hunk 4]
```diff
@@ -26,41 +26,46 @@
...
       "cell_type": "markdown",
       "metadata": {
-        "id": "ij53c3EzrKoS"
+        "id": "bc4b12e95217"
       },
       "source": [
         "## Overview\n",
         "\n",
-        "This notebook demonstrates how to create a Custom Model using Custom Python Package Training, with a Vertex AI Dataset, and how to serve the model using TensorFlow-Serving Container for online prediction, and batch prediction. It requires you to provide a bucket where the dataset will be stored.\n",
+        "This notebook demonstrates how to create a Custom Model using Custom Python Package Training, with a Vertex AI Dataset, and how to serve the model using TensorFlow-Serving Container for online prediction, and batch prediction. It requires you to provide a bucket to store the dataset.\n",
```
[the remark this answers is not in the document]
Comment (k-root) [reply notebooks/official/sdk/SDK_Custom_Training_Python_Package_Managed_Text_Dataset_Tensorflow_Serving_Container.ipynb]: Here. "It" is just the notebook/tutorial. 
I changed the sentence to "A GCS bucket is required here to store the dataset."

**Comment by k-root** [comment]
The notebook is failing lint check. but when the lint check is run on the same notebook in workbench instance, everything is passing with message "All tests finished. Exiting with return code = 0". Please suggest how to proceed

**Comment by katiemn** [comment]
Are you able to run the lint check locally?

**Comment by k-root** [comment]
> Are you able to run the lint check locally?

@katiemn , Yes. I am able to run both lint checks(python way and the docker way as well) locally. The lint check exits with the message "All tests finished. Exiting with return code = 0"

**Review by katiemn — APPROVED** [review]

**Pull request merged**